api/src/db/models/unverify.rs

```diff
@@ -61,3 +62,17 @@ pub struct Instruction {
     /// Inner instructions generated during execution
     pub innerInstructions: Vec<Value>,
 }
+
+/// Extracts and validates the upgrade instruction from the payload
+pub fn extract_instruction(
```

Comment:
can we rename this to `parse_helius_transaction`? The current name is quite confusing